src/student_success_tool/modeling/bias_detection.py

```diff
@@ -20,17 +25,278 @@
     "🔴 HIGH BIAS": "high_bias",
 }
 
+# TODO: eventually we should use the custom_style.mplstyle colors, but currently
+# the color palette does not have distinct enough colors for calibration by group
+# where there are a lot of subgroups
+PALETTE = sns.color_palette("Paired")
+
+
+def evaluate_bias(
+    run_id: str,
+    df_pred: pd.DataFrame,
+    split_col: str,
+    student_group_cols: list,
+    target_col: str,
+    pred_col: str,
+    pred_prob_col: str,
+    pos_label: str,
+) -> None:
```

Comment:
ah, I see that you're deliberately mirroring the API for `evaluate_performance()`. that's a nice idea! my comments over there also apply here, so I encourage you to tidy them up in the same ways :)